### DISCUSSION TITLE
Facility Managed by 3rd Party - Footer Requirements

### DISCUSSION THREAD

**Discussion by jzh512** [CMSgov/hospital-price-transparency#24] [Q&A] [answered]
One of our health system's facilities is managed by a 3rd party, and the MRF and Shoppable Services links are on that facility's homepage.  This 3rd party manages several other facilities not affiliated with our health system as well.  Since the MRF and Shoppable Services files are not on a separate site from this facility's homepage (i.e., not on the 3rd party's main site), do the footer requirements still apply even though the footer would just link to the same page?  Thank you.

**Comment by RidenourK** [comment, marked as answer]
@jzh512. Thank you for your question. 

As indicated in the CY 2024 OPPS/ASC Final Rule, at new § 180.50(d)(6)(ii), beginning January 1, 2024, the hospital must ensure that the public website it selects to host its machine-readable file establishes and maintains, in the form and manner specified by CMS: A link in the footer on its website, including but not limited to the homepage, that is labeled “Price Transparency” and links directly to the publicly available web page that hosts the link to the machine-readable file.

CMS remains committed to ensuring that the MRFs and their data contents are easily accessible and does not believe that offering flexibility on placement of the proposed footer link would achieve a predictable navigation path to internal web pages because link placement could vary from one hospital website to another. We believe the addition of standardized hyperlinks in the footer of hospital websites would aid in the automation of MRF data retrieval by creating a predictable navigation path to internal web pages that describe the HPT program and providing direct links to the MRF location….At minimum, the link in the footer must be on the homepage.

[Please refer to 88 FR 82113](https://www.federalregister.gov/d/2023-24293/p-5081).
